File: src/main/java/com/brian/Hello.java
package com.brian;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello Java");
        Person1 person1 = new Person1(70f, 1.75f);
        person1.weight = 70f;
        person1.high = 1.75f;
        System.out.println(person1.bmi());

//        int a = 123;
//        float b = 123.5f;
//        double c = 123.5;
//        char d = 'a';
//        boolean e = false;
//        String f = "d";
//        byte g = 3;
    }
}

File: src/main/kotlin/com/brian/Hello.kt
package com.brian

fun main(args: Array<String>) {
//    println("Hello Kotlin");
//    Person2().hello();
    val person2 = Person2(weight = 70f, high = 1.7f);
    println(person2.bmi());
//    person2.hello();
//    var a: Int = 1;
//    a = 2;

    val apple: Char = 'A';
    println(apple.toInt() > 80);

    var s : String? = "aaa";
    s = null;
    println(s!!.length);
}

class Person2(var name: String = "", var weight: Float, var high: Float) {

    fun bmi(): Float {
        return weight / (high * high);
    }

    fun hello() {
        println("Hello Kotlin");
    }
}